package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Learn how to use istanbul for tracking test/code coverage in your JS projects",
   "main": "test.js",
   "scripts": {
-    "coverage": "node ./node_modules/.bin/istanbul cover ./test.js",
+    "coverage": "istanbul cover ./test.js",
```

Comment:
Good point. However installing it globally is not the bad idea for its awesome CLI. 
We can actually mention it in docs too. I am happy to do another PR. Does it sound good?